.coveragerc

```diff
@@ -0,0 +1,16 @@
+[run]
+branch = True
+
+[report]
+exclude_lines =
+    # Don't complain about abstract methods, they aren't run:
+    @(abc\.)?abstractmethod
+
+omit =
+    src/prefect/orion/database/migrations/versions/*
+    tests/*
```

Comment:
Can we move this to the `pytest` invocation just for CI?  Reporting coverage information about tests helps to identify poor tests and those that are being "masked" like this:

```python
def test_this_super_awesome_thing():
  assert 1 == 2  # this test never runs

def test_this_super_awesome_thing():
  assert 1 == 1
```

I agree we wouldn't be that worried about this in CI, but it would be nice locally and I don't know if you can un-omit something on the CLI.

Reply:
Yeah, I'll work on that!

Reply:
@chrisguidry I couldn't find a way to do this cleanly, so for now I kept reporting on for tests. That wasn't my main concern here anyway!